packages/x-date-pickers/src/tests/describeJalaliAdapter/describeJalaliAdapter.ts

```diff
@@ -0,0 +1,34 @@
+import createDescribe from '@mui/monorepo/test/utils/createDescribe';
```

Comment:
Unlike on `date-io` I created shared tests for those two adapters
That way we can assert that they are working the same way, which reduces the risk of mistakes